typescript/cosmos-types/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@hyperlane-xyz/cosmos-types",
+  "version": "1.0.0",
+  "description": "Hyperlane TypeScript SDK types for the Cosmos Hyperlane SDK module",
+  "type": "module",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "types": "dist/index.d.ts",
+  "files": [
+    "/dist"
+  ],
+  "homepage": "https://www.hyperlane.xyz",
+  "repository": "https://github.com/hyperlane-xyz/hyperlane-monorepo",
+  "keywords": [
+    "hyperlane",
+    "cosmos",
+    "blockchain",
+    "sdk",
+    "types"
+  ],
+  "license": "Apache-2.0",
+  "scripts": {
+    "build": "rimraf dist && tsc",
+    "format": "prettier --write .",
+    "lint": "eslint -c ./eslint.config.mjs",
+    "prettier": "prettier --write ./src",
+    "clean": "rm -rf ./dist ./cache",
+    "test": "exit 0"
```

Comment:
is there anything we can test? maybe add a test to check that if we regenerate types in CI that we don't get a diff?

Reply:
even if we don't test anything, would be good to add an echo "no tests in cosmos-sdk-types" or something to log that we do try this

Reply:
unfortunately there is nothing to test since all is autogenerated and just exported, will echo `no tests in cosmos-types`